src/Compilers/Core/Portable/Operations/IInvalidExpression.cs

```diff
@@ -13,7 +13,7 @@ public interface IInvalidExpression : IOperation
         /// <summary>
         /// Child operations.
         /// </summary>
-        ImmutableArray<IOperation> Children { get; }
```

Comment:
We're making changes in the other direction in other parts of the code (preferring ImmutableArray over IEnumerable, when we have an ImmutableArray at hand).
[https://github.com/dotnet/roslyn/pull/20610](https://github.com/dotnet/roslyn/pull/20610)